core/models.py

```diff
@@ -577,8 +577,6 @@ def save(self, *args, **kwargs):
            #save the new object
```

Comment:
Can you confirm that django will understand self.id without aliases like id = AliasField(db_column='uuid') ?

Reply:
Ok, I want to ask you question to be sure about "id" vs "uuid". You want to replace normal, standard "id" (integer value) into "uuid", don't you? For example there will be only uuid as a primary key and "id" should be the same as uuid. Is it exactly what you want? If yes, I'm gonna to fix it so as to have only uuid/id as a uuid. @dborowiecki mentioned to me earlier about this problem and we were wondering whether should be only "uuid" as "uuid" or "id/uuid" as a "uuid" or "id" as an integer.